### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by aniketxparihar**
Added Signup Page Please Review!
image

### CODE REVIEW

File: signup/signup.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                </i></a>
+                <div class="hidden__link pointer close"><i
+                    class="badge__icon material-icons relative p-5 txt-gray-400">close
+                </i></div>
+    </div>
+
+    <main>
+
+        <div id="form__container">
+            <div id="heading">Signup</div>
```
Comment (Pratik1005) [inline signup/signup.html]: You can use heading tag for titles for accessibility.

File: signup/signup.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            <form action="">
```
Comment (Pratik1005) [inline signup/signup.html]: remove the action attribute if not using

**Review by Pratik1005 — APPROVED** [review]
I have added review rest looks good

**Review by xsaurabhsuthar — APPROVED** [review]

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (anjali1102) [inline .vscode/settings.json:2]: suggestions: put in gitignore

**Review by anjali1102 — APPROVED** [review]

**Pull request merged**